assets/app/scripts/services/navigate.js

```diff
@@ -12,7 +12,7 @@ angular.module("openshiftConsole")
        */
       toErrorPage: function(message, errorCode) {
         var redirect = URI('error').query({
-          error_description: message,
+          'error_description': message,
```

Comment:
Yeah, this is the camelCase deal (errorDescription vs error_description), I think it is deprecated in jshint since its more about consistent style than an error.  We could probably put it back and ignore...